Src/GoogleApis.Auth/OAuth2/UserCredential.cs

```diff
@@ -91,17 +91,9 @@ public UserCredential(IAuthorizationCodeFlow flow, string userId, TokenResponse
         /// minute away from expiration. If token server is unavailable, it will try to use the access token even if 
         /// has expired. If successful, it will call <see cref="IAccessMethod.Intercept"/>.
         /// </summary>
-        public async Task InterceptAsync(HttpRequestMessage request, CancellationToken taskCancellationToken)
+        public async Task InterceptAsync(HttpRequestMessage request, CancellationToken cancellationToken)

```

Comment:
Why do you need this renaming, let's try to keep not necessary changes out of this one.
By the way, I wanted to have "taskCancellationToken", since it might be confusing "token"... is it access token? refresh token? what is cancellation token.. so I just added task as prefix...